collect_app/src/main/java/org/odk/collect/android/database/DatabaseFormsRepository.java

```diff
@@ -48,6 +48,15 @@ public DatabaseFormsRepository() {
         }
     }
 
+    @Override
+    public List<Form> getAll(String jrFormId, @Nullable String jrVersion) {
```

Comment:
I'm not sure about naming, here. There's `getByJrFormIdNotDeleted` above. But I can't think of a case where we'd want deleted forms so calling it `getAll` seemed right to me.